autoload/netrw.vim

```diff
@@ -8705,8 +8705,7 @@ function! s:NetrwLocalListingList(dirname,setmaxfilenamelen)
     " get the list of files contained in the current directory
     let dirname    = a:dirname
     let dirnamelen = strlen(dirname)
-    let filelist   = netrw#fs#Glob(dirname,"*",0)
-    let filelist   = filelist + netrw#fs#Glob(dirname,".*",0)
+    let filelist   = map(['.', '..'] + readdir(dirname), 'dirname . "/" . v:val')
```

Comment:
Use `netrw#fs#JoinPath()` instead of harcoding '/'

Reply:
All right, fixed.